### PR TITLE
Test for trace list

### PR DESCRIPTION
**Pull request by imjfckm**

### CODE REVIEW

File: tests/functional/test_trace_management.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                    f"Wrong trace label. Expected: {exp_label}, got: {trace['label']}"
+                )
+
+            exp_state = "COMPLETE" if exp_label in expected_labels[:-1] else "RUNNING"
+            if trace["state"] != exp_state:
+                TestRun.LOGGER.error(
+                    f"Wrong trace state. Expected: {exp_state}, got: {trace['state']}"
+                )
+
+    with TestRun.step("Check listing by prefix"):
```
Comment (mmichal10) [inline tests/functional/test_trace_management.py]: How about adding to list trace with prefix different than 'kernel'?

Comment (imjfckm) [reply tests/functional/test_trace_management.py]: Is `iotrace` actually able to produce other than kernel stats right now?

File: tests/utils/iotrace.py [hunk 2]
```diff
@@ -163,7 +163,7 @@ def get_traces_list(prefix: str = None, shortcut: bool = False) -> list:
             command += (' -p ' if shortcut else ' --prefix ') + f'{prefix}'
         output = TestRun.executor.run_expect_success(command)
 
-        return parse_json(output.stdout)
+        return parse_json(output.stdout)[0]["trace"]
```
Comment (Ostrokrzew) [inline tests/utils/iotrace.py]: Please, update the use of this method, especially in the next one.

Comment (imjfckm) [reply tests/utils/iotrace.py]: Fixed!

File: tests/functional/test_trace_management.py (hunk 1 above)
Comment (mariuszbarczak) [reply tests/functional/test_trace_management.py]: It's not

**Comment by mariuszbarczak** [comment]
@imjfckm please remove conflicts

**Pull request merged**